File: Programmers/12930.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    bool isEven = true;
    for(int i = 0; i < s.length(); i++) {
        if(s[i] == ' ') {
            answer += " ";
            isEven = true;
            continue;
        }
            
        answer += isEven ? toupper(s[i]) : tolower(s[i]);
        isEven = !isEven;
    }
    
    return answer;
}

File: Python/12930.py
def solution(s):
    answer = ''
    cnt = 0
    for i in range(len(s)):
        if s[i] == ' ':
            cnt = 0
            answer += ' '
            continue
        if cnt % 2 == 1:
            answer += s[i].lower()
        else:
            answer += s[i].upper()
        cnt += 1

    return answer